package.json

```diff
@@ -24,6 +24,7 @@
     "test-filter-parsing-errors": "./node_modules/.bin/tape tests/test-filter-parsing-errors.js | ./node_modules/.bin/faucet"
```

Comment:
I'd like to have a `"test-filter-when-format"` command so we can test things separately on development.